examples/gocean/eg5/Makefile

```diff
@@ -98,6 +98,13 @@ dl_timer: PROFILE_LINK=-ldl_timer_psy -L$(DL_TIMER_ROOT) -ldl_timer_omp -fopenmp
 dl_timer: F90FLAGS += -fopenmp   # apeg-dl_timer must be compiled with openmp!
 dl_timer: $(NAME).dl_timer
```

Comment:
I'm sure we've discussed this before but is there some good reason why doing e.g. `make lfric` doesn't build the wrapper library for the lfric timer?

Reply:
This is tracked in #917, am happy to look at this once this is merged.